scripts/setup_kubernetes.sh

```diff
@@ -13,6 +13,6 @@ tar xf doctl-1.14.0-linux-amd64.tar.gz
 chmod +x ./doctl
 sudo mv ./doctl /usr/local/bin/doctl
 
-doctl auth init --access-token $DOCTL_ACCESS_TOKEN
+doctl auth --context develop init --access-token $DIGITALOCEAN_ACCESS_TOKEN
```

Comment:
Yes, so much better! I haven't really understood the documentation of the `context` though. https://github.com/digitalocean/doctl I don't know the use case of it.

Reply:
it allows you to use multiple digital ocean accounts
https://github.com/digitalocean/doctl#logging-in-to-multiple-digitalocean-accounts